server/tests/backends/fixtures/filter/dates_from_until_pypika.json

```diff
@@ -0,0 +1,103 @@
+{
+  "description": "bigquery behaves differently on the cast !",
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake",
+    "bigquery_pypika"
```

Comment:
didn't succeed handling bigquery 🤔  yet !
it returns no errors but 0 results !

Reply:
Found a workaround here : https://github.com/ToucanToco/weaverbird/pull/1413/commits/f03f7574f558c938eec7009d95231e0edce0956e